test/run-broken.sh

```diff
@@ -15,14 +15,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -e
 set -o verbose
-rm -rf /tmp/ops
+bundle exec chef-zero &
+ZERO=$!
+trap "kill -9 $ZERO" EXIT
+SOURCE=/tmp/ops
```

Comment:
This should probably use `mktemp -d` for safety

Reply:
So the scripts that this calls (ie: `tt-auto.rb` and `tt-git.rb`) are referencing the specific repo `/tmp/ops` and while I can probably feed the temp dir to this, I'm not sure that I want to tackle that here too.